lib/puma_patch.rb

```diff
@@ -0,0 +1,18 @@
+
+module PumaPatch
+  def mount(path, srv, *options)
+    require 'rack/lobster'
+    Rack::Builder.new do
+      map path do
+        run srv
+      end
+    end
+  end
+
+  def start
+    self.run.join
+  end
+end
+
+require 'puma'
+Puma::Server.class_eval { include PumaPatch }
```

Comment:
I don't like monkey patching just because of one or two simple calls. In the method where you call `mount` you already have `case server_name` anyway, I suggest to do the mapping there, or in a launcher helper.

Reply:
ack